api/src/resolvers/campaigns/campaigns-resolver.ts

```diff
@@ -110,17 +110,17 @@ export const campaignsMutation = {
       const startDate = new Date(equipmentCampaign.campaign?.equipmentStartDate)
       const newStartDate = new Date(args.startDate)
 
+      const setEquipmentDuration = rearrangeCypherObject(
+        await session.run(SetEquipmentDeadline, args)
```

Comment:
no error handling here

Reply:
It's wrapped in a try catch. The try starts earlier